src/less/webmakerLogin.less

```diff
@@ -361,7 +359,14 @@
         }
       }
       input[type="checkbox"] {
-        display: none;
+        position: absolute;
+        left: -9000px;
+      }
+      input[type="checkbox"]:focus + label {
+        div {
+            border: 2px solid #E1EBF1;
+            box-shadow: 5px 5px 5px 0px rgba(0,0,0,0.67);
```

Comment:
I think you might want to change this to match the above style in terms of sizing, but what do you think @vazquez ?